website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -32,7 +32,7 @@ The `Bubble` element is a pure container that yields the children inside the "po
 
 !!! Info
 
-Note: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset, please [speak with the Design System team](/about/support).
+Note: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset, [contact the Design Systems team](/about/support).
```

Comment:
```suggestion
Note: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset, [contact the Design Systems Team](/about/support).
```